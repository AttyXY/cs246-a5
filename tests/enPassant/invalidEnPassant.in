game human human
move e2 e4
move d7 d5
move f2 f4
move d5 d4
move g2 g4
move d4 e3
move h7 h5
move c2 c4
move d4 c3
