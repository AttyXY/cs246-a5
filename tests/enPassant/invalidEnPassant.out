WELCOME TO CHESS! 
PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]
- quit (to exit)

STARTING GAME!

8  rnbqkbnr
7  pppppppp
6  --------
5  --------
4  --------
3  --------
2  PPPPPPPP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  pppppppp
6  --------
5  --------
4  ----P---
3  --------
2  PPPP-PPP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  ppp-pppp
6  --------
5  ---p----
4  ----P---
3  --------
2  PPPP-PPP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  ppp-pppp
6  --------
5  ---p----
4  ----PP--
3  --------
2  PPPP--PP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  ppp-pppp
6  --------
5  --------
4  ---pPP--
3  --------
2  PPPP--PP
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  ppp-pppp
6  --------
5  --------
4  ---pPPP-
3  --------
2  PPPP---P
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  ppp-pppp
6  --------
5  --------
4  ---pPPP-
3  --------
2  PPPP---P
1  RNBQKBNR
   abcdefgh

Invalid move. Input another move.
8  rnbqkbnr
7  ppp-ppp-
6  --------
5  -------p
4  ---pPPP-
3  --------
2  PPPP---P
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  ppp-ppp-
6  --------
5  -------p
4  --PpPPP-
3  --------
2  PP-P---P
1  RNBQKBNR
   abcdefgh

8  rnbqkbnr
7  ppp-ppp-
6  --------
5  -------p
4  ----PPP-
3  --p-----
2  PP-P---P
1  RNBQKBNR
   abcdefgh


PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO CONTINUE:
- setup (to create new setup)
- game (to continue with previous setup)
- quit (to exit)

THANKS FOR PLAYING CHESS!
Final Score:
White: 0
Black: 0
